cmd/runtimetest/main.go

```diff
@@ -81,32 +81,11 @@ func loadSpecConfig(path string) (spec *rspec.Spec, err error) {
 	return spec, nil
 }
 
-// should be included by other platform specified process validation
-func validateProcess(spec *rspec.Spec) error {
-	if spec.Process.Cwd != "" {
-		cwd, err := os.Getwd()
-		if err != nil {
-			return err
-		}
-		if cwd != spec.Process.Cwd {
-			return fmt.Errorf("Cwd expected: %v, actual: %v", spec.Process.Cwd, cwd)
-		}
-	}
-
-	for _, env := range spec.Process.Env {
-		parts := strings.Split(env, "=")
-		key := parts[0]
-		expectedValue := parts[1]
-		actualValue := os.Getenv(key)
-		if actualValue != expectedValue {
-			return fmt.Errorf("Env %v expected: %v, actual: %v", key, expectedValue, actualValue)
-		}
+func validateUser(spec *rspec.Spec) error {
+	if runtime.GOOS == "windows" {
+		return nil
```

Comment:
Perhaps this function should be `validatePOSIXUser` to [match the spec][1] and then used in both `linuxValidatations` and a new `solarisValidations`?  Or a new `posixValidations` which is included for both `linux` and `solaris` platforms?  The spec *has* [Windows-specific user stuff][2], but all of its RFC 2119 language is currently config-oriented and covered by the JSON Schema validation.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/config.md#posix-platform-user
[2]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/config.md#windows-user

Reply:
>Or a new posixValidations which is included for both linux and solaris platforms?

I think it would be better. I'm going to do that.

Reply:
updated, PTAL.